files/default/sql/ddl/2.1.0__initial_tables.sql

```diff
@@ -1032,6 +1031,24 @@ CREATE TABLE `project` (
 ) ENGINE=ndbcluster AUTO_INCREMENT=119 DEFAULT CHARSET=latin1 COLLATE=latin1_general_cs;
 /*!40101 SET character_set_client = @saved_cs_client */;
 
+--
+-- Table structure for table `python_environment`
+--
+
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!40101 SET character_set_client = utf8 */;
+CREATE TABLE `python_environment` (
+  `id` int(11) NOT NULL AUTO_INCREMENT,
+  `project_id` int(11) NOT NULL,
+  `python_version` VARCHAR(25) COLLATE latin1_general_cs NOT NULL,
+  `jupyter_conflicts` TINYINT(1) NOT NULL DEFAULT '0',
+  `conflicts` VARCHAR(12000) COLLATE latin1_general_cs DEFAULT NULL,
+  UNIQUE KEY `project_env` (`project_id`),
+  PRIMARY KEY (`id`),
+  CONSTRAINT `FK_PROJECT_ID` FOREIGN KEY (`project_id`) REFERENCES `project` (`id`) ON DELETE CASCADE ON UPDATE NO ACTION
```

Comment:
```suggestion
  CONSTRAINT `FK_PYTHONENV_PROJECT` FOREIGN KEY (`project_id`) REFERENCES `project` (`id`) ON DELETE CASCADE ON UPDATE NO ACTION
```
just to make it a bit more unique

Reply:
fixed